### PR TITLE
fix: adding return to cashier closing

### PR DESCRIPTION
**Pull request by elba7r**
this pull request based on @sagarvora request due to conversation on this pull request ( adding returns field to cashier closing #15752 )

### CODE REVIEW

File: erpnext/accounts/doctype/cashier_closing/cashier_closing.json [hunk 1]
```diff
@@ -20,7 +20,440 @@
    "allow_on_submit": 0, 
    "bold": 0, 
    "collapsible": 0, 
-   "columns": 0, 
+   "columns": 0, {
```
Comment (sagarvora) [inline erpnext/accounts/doctype/cashier_closing/cashier_closing.json]: Incorrect JSON format. Have you manually modified this file?

**Review by sagarvora — CHANGES_REQUESTED** [review]

File: erpnext/accounts/doctype/cashier_closing/cashier_closing.json (hunk 1 above)
Comment (elba7r) [reply erpnext/accounts/doctype/cashier_closing/cashier_closing.json]: yes, but i think i fixed it now.

Comment (sagarvora) [reply erpnext/accounts/doctype/cashier_closing/cashier_closing.json]: Please don't modify it manually. The `modified` timestamp must be auto-generated. You can edit the DocType.

Comment (elba7r) [reply erpnext/accounts/doctype/cashier_closing/cashier_closing.json]: ok, i got it but i see there is no conflicts or errors by the json file so, u can merge it ! or i must re fix it ?

Comment (sagarvora) [reply erpnext/accounts/doctype/cashier_closing/cashier_closing.json]: It must be fixed in order to correctly update database while migration. Please make changes using the doctype. Thanks.

Comment (elba7r) [reply erpnext/accounts/doctype/cashier_closing/cashier_closing.json]: ok, i hope its fixed now

Comment (elba7r) [reply erpnext/accounts/doctype/cashier_closing/cashier_closing.json]: now i fixed it to be correct with the current version in develop branch + adding  the returns, do i need to make any other changes, or its ready for accept?

**Review by sagarvora — APPROVED** [review]

**Pull request merged**